CHANGELOG.md

```diff
@@ -13,11 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-- Handle messages in openbrush-style traits (PSP22::transfer, etc.)
+- Handle messages in openbrush-style traits (`PSP22::transfer`, etc.)
 
 ### Changed
 
 - ink-wrapper-types released to crates.io
+- Return a custom type that implements `std::error::Error` instead of `ink_primitives::LangError`
```

Comment:
if we're putting everything in backticks now then maybe `ink-wrapper-types` in the line above as well?

Reply:
:+1: